src/components/Pages/OfficialDocuments/_OfficialDocument.scss

```diff
@@ -4,7 +4,7 @@
   margin-top: 3rem;
 
   &:not(:last-child) {
-    border-bottom: 1.5px solid $coa-color-gray-lighter;
+    border-bottom: $coa-border-bottom-small-light-gray;
```

Comment:
ooh! I may need to start using this one

Reply:
lol. The name really is something, isn't it!